### PR TITLE
[site] Add license request success page

### PR DESCRIPTION
**Pull request by ToyTox**

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/pages/license_request_success_ru.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+layout: page-nosidebar
+lang: ru
+toc: false
+---
+
+<p>Спасибо за ваш интерес к платформе Deckhouse Kubernetes Platrofm!</p>
+<p>Ваш токен для доступа к EE-редакции платформы (мы также отправили его вам на почту):</p>
+<p id="license-token" style="font-size: 21px; font-weight: 600; letter-spacing: -0.4px; color: #0066FF;"></p>
+<p>Лицензия будет действовать на протяжении одного месяца, а для её продления потребуется связаться с командой Deckhouse (<a href="mailto:sales@deckhouse.io">sales@deckhouse.io</a>).</p>
+<p>Вот инструкции для начала использования Deckhouse: <a href="https://deckhouse.io/ru/gs">https://deckhouse.io/ru/gs</a></p>
```
Comment (z9r5) [inline docs/site/pages/license_request_success_ru.html:14]:
```suggestion
<p>Вот инструкции для начала использования Deckhouse: <a href="https://deckhouse.io/ru/gs">https://deckhouse.io/ru/gs/</a></p>
```

File: docs/site/pages/license_request_success.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+permalink: en/license_request_success.html
+layout: page-nosidebar
+toc: false
+---
+
+<p>Thank you for your interest in the Deckhouse Kubernetes platform!</p>
+<p>Below is your token for accessing the EE edition of the platform (we also sent it to your email):</p>
+<p id="license-token" style="font-size: 21px; font-weight: 600; letter-spacing: -0.4px; color: #0066FF;"></p>
+<p>Note that the license is valid for one month. Please, contact the Deckhouse team (<a href="mailto:sales@deckhouse.io">sales@deckhouse.io</a>) to renew it.</p>
+<p>The Deckhouse Getting Started guide is available here: <a href="https://deckhouse.io/en/gs">https://deckhouse.io/en/gs</a>.</p>
```
Comment (z9r5) [inline docs/site/pages/license_request_success.html:13]:
```suggestion
<p>The Deckhouse Getting Started guide is available here: <a href="https://deckhouse.io/en/gs">https://deckhouse.io/en/gs/</a>.</p>
```

File: docs/site/pages/license_request_success.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: One month trial
+description: One month trial
+permalink: en/license_request_success.html
+layout: page-nosidebar
+toc: false
```
Comment (z9r5) [inline docs/site/pages/license_request_success.html:6]:
```suggestion
toc: false
searchable: false
sitemap_include: false
```

File: docs/site/pages/license_request_success_ru.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: Тестовая лицензия на один месяц
+description: Тестовая лицензия на один месяц
+permalink: ru/license_request_success.html
+layout: page-nosidebar
+lang: ru
```
Comment (z9r5) [inline docs/site/pages/license_request_success_ru.html:6]:
```suggestion
lang: ru
searchable: false
sitemap_include: false
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/pages/license_request_success_ru.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+searchable: false
+sitemap_include: false
+toc: false
+---
+
+<p>Спасибо за ваш интерес к платформе Deckhouse Kubernetes Platrofm!</p>
+<p>Ваш токен для доступа к EE-редакции платформы (мы также отправили его вам на почту):</p>
+<p id="license-token" style="font-size: 21px; font-weight: 600; letter-spacing: -0.4px; color: #0066FF;"></p>
+<p>Лицензия будет действовать на протяжении одного месяца, а для её продления потребуется связаться с командой Deckhouse (<a href="mailto:sales@deckhouse.io">sales@deckhouse.io</a>).</p>
+<p>Вот инструкции для начала использования Deckhouse: <a href="https://deckhouse.io/ru/gs/">https://deckhouse.io/ru/gs/</a></p>.
```
Comment (z9r5) [inline docs/site/pages/license_request_success_ru.html:16]:
```suggestion
<p>Вот инструкции для начала использования Deckhouse: <a href="https://deckhouse.io/ru/gs/">https://deckhouse.io/ru/gs/</a>.</p>
```

**Pull request merged**